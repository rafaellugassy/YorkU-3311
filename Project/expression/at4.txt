forall
start_of_set_enumeration 
conjunction
bool_value (true)
bool_value (true)
disjunction
bool_value (true)
bool_value (false)
exclusive_or -- true xor true is false
bool_value (true)
bool_value (true)
end_of_set_enumeration
analyze
simplify 