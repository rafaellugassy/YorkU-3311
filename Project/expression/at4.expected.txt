  Expression currently specified: ?
  Report: Expression is initialized.
->forall
  Expression currently specified: (&& ?)
  Report: OK.
->start_of_set_enumeration
  Expression currently specified: (&& {?})
  Report: OK.
->conjunction
  Expression currently specified: (&& {(? && nil)})
  Report: OK.
->bool_value(True)
  Expression currently specified: (&& {(True && ?)})
  Report: OK.
->bool_value(True)
  Expression currently specified: (&& {(True && True), ?})
  Report: OK.
->disjunction
  Expression currently specified: (&& {(True && True), (? || nil)})
  Report: OK.
->bool_value(True)
  Expression currently specified: (&& {(True && True), (True || ?)})
  Report: OK.
->bool_value(False)
  Expression currently specified: (&& {(True && True), (True || False), ?})
  Report: OK.
->exclusive_or
  Expression currently specified: (&& {(True && True), (True || False), (? xor nil)})
  Report: OK.
->bool_value(True)
  Expression currently specified: (&& {(True && True), (True || False), (True xor ?)})
  Report: OK.
->bool_value(True)
  Expression currently specified: (&& {(True && True), (True || False), (True xor True), ?})
  Report: OK.
->end_of_set_enumeration
  Expression currently specified: (&& {(True && True), (True || False), (True xor True)})
  Report: OK.
->analyze
  Expression currently specified: (&& {(True && True), (True || False), (True xor True)})
  Report: (&& {(True && True), (True || False), (True xor True)}) is type-correct.
->simplify
  Expression currently specified: (&& {(True && True), (True || False), (True xor True)})
  Report: False
